pkg/proto/nbcontract/v1/gpuconfig.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package nbcontract.v1;
+
+import "pkg/proto/nbcontract/v1/enum.proto";
+
+message GpuConfig {
+  // nvidia_state represents the Nvidia GPU state
```

Comment:
I don't quite understand the meaning of this, at least not from the comment. The desired state of Nvidia GPU on the node, to be set by the bootstrap process?

Related: Do we have a single place that captures all of the different GPU settings we support, with their meaning and interaction? Still concisely, but longer form than field comments? (It would also capture the relationship between these and the NVidia operator ...)

Reply:
For the related question: there's a PRD for GPU operator which captures constraints and certain config relationships (limited though). It would be a good to-do for us to add.

Reply:
@ganeshkumarashok That will be awesome. We will add the detailed descriptions back once they're ready.